common/changes/@itwin/presentation-common/deprecate-presentation-rpc_2024-11-04-15-22.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-common",
+      "comment": "deprecate PresentationRpcInterface",
+      "type": "none"
+    }
```

Comment:
```suggestion
    {
      "packageName": "@itwin/presentation-common",
      "comment": "All public methods of `PresentationRpcInterface` have been deprecated. Going forward, RPC interfaces should not be called directly. Public wrappers such as `PresentationManager` should be used instead.",
      "type": "minor"
    }
```

Reply:
I'm not totally sure about `minor`, but the deprecation should definitely be mentioned in the changelog. Chose `minor` over `patch`, because consumers will start getting deprecation warnings, if they're using these apis.

Reply:
Does the type even matter here? The version will be bumped in lock step anyway

Reply:
Ah, right. As long as it's not "none", it probably doesn't matter.